assets/airports/rjss.json

```diff
@@ -0,0 +1,960 @@
+                ["N38d14m08.01", "E140d49m01.20"],
+                ["N38d13m41.87", "E140d49m02.63"],
+                ["N38d13m15.92", "E140d49m06.96"],
+                ["N38d12m50.36", "E140d49m14.16"],
+                ["N38d12m36.00", "E140d49m20.00"],
+                ["N38d12m16.00", "E141d01m20.00"],
+                ["N37d55m55.00", "E141d49m37.00"],
+                ["N37d18m49.00", "E141d03m52.00"],
+                ["N37d18m30.27", "E141d00m49.68"],
+                ["N37d18m18.86", "E140d55m17.40"],
+                ["N37d18m30.27", "E140d49m45.11"],
+                ["N37d19m04.44", "E140d44m15.00"],
+                ["N37d19m27.00", "E140d42m07.00"],
+                ["N37d20m01.85", "E140d41m44.87"],
+                ["N37d20m40.16", "E140d41m17.87"],
+                ["N37d20m47.00", "E140d41m13.00"],
+                ["N37d27m37.00", "E140d43m58.00"],
+                ["N37d28m46.77", "E140d42m22.31"],
+                ["N37d29m50.49", "E140d40m37.91"],
+                ["N37d30m46.76", "E140d38m47.03"],
+                ["N37d31m35.07", "E140d36m50.40"],
+                ["N37d32m15.14", "E140d34m49.07"],
+                ["N37d32m46.64", "E140d32m43.44"],
+                ["N37d33m09.28", "E140d30m35.28"],
+                ["N37d33m22.93", "E140d28m24.96"],
+                ["N37d33m24.00", "E140d27m51.00"],
+                ["N38d09m11.00", "E140d27m51.00"],
+                ["N38d09m13.32", "E140d28m01.55"],
+                ["N38d09m43.34", "E140d29m34.07"],
+                ["N38d10m19.59", "E140d31m03.00"],
+                ["N38d11m01.78", "E140d32m27.23"],
+                ["N38d11m49.59", "E140d33m46.79"],
+                ["N38d12m42.62", "E140d35m00.59"],
+                ["N38d13m40.51", "E140d36m08.28"],
+                ["N38d14m42.82", "E140d37m09.48"],
+                ["N38d15m49.03", "E140d38m03.11"],
+                ["N38d16m58.69", "E140d38m49.55"],
+                ["N38d18m11.19", "E140d39m27.72"],
+                ["N38d19m23.00", "E140d39m55.00"]
+            ]
+        }
+    ],
+    "fixes": {
+        "_IXE024047"   : ["N37d51m32.00", "E141d22m34.00"],
+        "_IXE024028"   : ["N37d34m29.00", "E141d12m54.00"],
+        "_IXE024SDE120": ["N37d54m56.00", "E141d24m31.00"],
+        "_RWY0901DME"  : ["N38d08m09.00", "E140d52m10.00"],
+        "_RWY0902DME"  : ["N38d08m07.00", "E140d51m51.00"],
+        "_RWY2702DME"  : ["N38d08m39.00", "E140d57m05.00"], 
+        "_RWY2703DME"  : ["N38d08m54.00", "E140d59m29.00"],
+        "_SDE080003"   : ["N38d08m54.00", "E140d59m32.00"],
+        "_SDE135010"   : ["N38d01m15.00", "E141d04m15.00"],
+        "_SDE120010"   : ["N38d03m19.00", "E141d06m17.00"],
+        "_SDE120005"   : ["N38d05m49.00", "E141d00m47.00"],
+        "_SDE105010"   : ["N38d05m43.00", "E141d07m33.00"],
+        "_SDE089010"   : ["N38d08m29.00", "E141d07m59.00"],
+        "_SS701233001" : ["N38d07m25.00", "E140d49m34.00"],
+        "ADNAP"        : ["N37d11m51.00", "E145d39m59.00"],
```

Comment:
Colons should only be aligned for visible fixes, so 5-letter fixes should have no space before the colons, and 3-letter VORs should have two spaces. Invisible fixes need not be aligned